### PR TITLE
S3C-703: update werelogs config handling

### PR DESCRIPTION
**Pull request by DavidPineauScality**
See issue for details

### CODE REVIEW

**Comment by DavidPineauScality** [comment]
@ironman-machine after=[scality/werelogs#99,scality/arsenal#314]

File: lib/shell.js [hunk 1]
```diff
@@ -5,11 +5,13 @@ const readline = require('readline');
...
 const client = new RESTClient([host]);
-const Logger = require('werelogs').Logger;
-const logging = new Logger('shell', {
+const werelogs = require('werelogs');
+
+werelogs.configure({
     level: 'info',
     dump: 'error',
 });
+const logging = new werelogs.Logger('shell');
```
Comment (michael-zapata) [inline lib/shell.js]: Shouldn't we call that `mdclient` instead?

Comment (DavidPineauScality) [inline lib/shell.js]: Renamed into `BucketClientShell` for consistency with REST Client, and show that it's the module handling the cli commands.

**Comment by DavidPineauScality** [comment]
Merging manually since E2E succeeded for all related PRs of the issue S3C-703:

E2E launched from: https://github.com/scality/S3/pull/852#issuecomment-321001002
E2E reported successs at: https://github.com/scality/S3/pull/852#issuecomment-321015792

**Pull request merged**